src/core/fileService.ts

```diff
@@ -48,8 +48,9 @@ interface ServiceOption {
   };
   ignore: string[];
   ignoreFile: string;
-  remoteExplorer?: {
+  remoteExplorer: {
```

Comment:
@Natizyskunk 
This value is used for comparison in the file `src/modules/remoteExplorer/treeDataProvider.ts`, so the default is required.
It can specify default values while comparing, but I think it is more inefficient.

Reply:
@NyaPPuu,
have you tried it to make sure it don't break anything ?

Reply:
Yes, I have been using the test version with this change since May when this PR was opened, and there has never been a problem.

Reply:
@NyaPPuu,
Perfect, seems good to me.